code/datums/repositories/crew/crew.dm

```diff
@@ -49,19 +49,15 @@ var/global/datum/repository/crew/crew_repository = new()
 
 	cache_data_alert[num2text(z_level)] = FALSE
 	var/tracked = scan()
-	for(var/obj/item/clothing/under/C in tracked)
```

Comment:
This was meant to be part of the vital monitor module, not the rig itself.
Everything should be based around that module, and the buttons for it should be in the module's section of the menu if possible